.spelling

```diff
@@ -1,6 +1,16 @@
+Bodgenhurst
+Brokenhurst
```

Comment:
Brokenhurst -> Brockenhurst

There are few instances where this incorrect spelling is used in the text; rather than attempting to find and point them all out individually, I'll just let the spell checker find them :)

Reply:
Somehow I miss
ed this one... 5b85ec8